mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrderById = (req, res) => {
+  Order.deleteOne({ _id: req.params.orderId })
+    .exec()
+    .then(result => {
+      if (result.deletedCount === 0) {
+        return res.status(404).send({ error: "No order found with that ID" });
+      }
+      res.status(200).send({ message: "Successfully deleted the order" });
+    })
+    .catch(err => {
```

Comment:
Inconsistent error handling. The catch block is incomplete and does not handle the error properly. It should be completed with a proper error handling mechanism, such as logging the error or sending an error response.